tools/modules/framework/manuals_data.sh

```
@@ -0,0 +1,52 @@
+module_options+=(
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)